fillers/withdrawals/withdrawals.py

```diff
@@ -492,7 +498,7 @@ def test_newly_created_contract(_):
 
     # Same test but include value in the contract creating transaction
 
-    tx.value = ONE_GWEI
+    block = block.with_fields(txs=[tx.with_fields(value=ONE_GWEI)])
```

Comment:
I feel like this should not be necessary. I see the use of `with_fields` when creating multiple similar txs that are going to be yielded to the same test, but once the yield has happened, it's like a clean slate, and the changes should never affect an already yielded test.